start

```diff
@@ -0,0 +1,331 @@
+  echo "                                file."
+  echo "    --multi_queue               Uses the multiqueue.override.yml compose file."
+  echo "    --nfs                       Uses the nfs.override.yml compose file."
+  echo "    --traefik                   Uses the traefik.override.yml compose file."
+  echo "    --use-external-database     Do not use postgres.override.yml compose file."
+  echo "    --use-external-broker       Do not use rabbitmq.override.yml compose file."
+  echo "    --flower                    Uses the flower.override.yml compose file."
+  echo "    --custom                    Uses custom.override.yml to leverage your"
+  echo "                                customized configuration."
+  echo "    --debug-build               See more verbose output from the build."
+  echo "    --elastic                   This spins up Elasticsearch and Kibana on your"
+  echo "                                machine (might need >=16GB of RAM)."
+  echo "    --https                     This leverage the https.override.yml file that"
+  echo "                                can be used to host IntelOwl with HTTPS and your"
+  echo "                                own certificate."
+  echo "    --pycti-version <value>     The pycti version to choose. This must match the"
+  echo "                                OpenCTI server version you are connecting to."
+  echo "                                Default is ${pycti_default_version}."
+  echo "    --pyelastic-version <value> The py-elasticsearch version to choose. This"
+  echo "                                must match the server version you are"
+  echo "                                connecting to. Default is ${pyelastic_default_version}."
+}
+
+check_parameters () {
+  if [[ $# == 1 && ( $1 == "--help" || $1 == "-h" ) ]]; then
+    print_help
+    exit 0
+  fi
+  if [[ $# -lt 2 ]]; then
+    echo "Error! Provide at least the environment and a command." >&2
+    print_synopsis
+    exit 2
+  fi
+
+  # little trick for exact matching in arrays
+  declare -A env_arguments=(["prod"]=1 ["test"]=1 ["ci"]=1)
+  declare -A test_mode=(["test"]=1 ["ci"]=1)
+  declare -A cmd_arguments=(["build"]=1 ["up"]=1 ["start"]=1 ["restart"]=1 ["down"]=1 ["stop"]=1 ["kill"]=1 ["logs"]=1 ["ps"]=1)
+
+  if ! [[ ${env_arguments["${1}"]} ]]; then
+    echo "Error! Illegal environment specified." >&2
+    exit 2
+  fi
+  if ! [[ ${cmd_arguments["${2}"]} ]]; then
+    echo "Error! Illegal command specified." >&2
+    exit 2
+  fi
+
+  env_argument=$1
+  cmd_argument=$2
+}
+
+set_defaults_values () {
+  project_name="intel_owl"
+  version=$current_version
+  export PYELASTIC_VERSION=$pyelastic_default_version
+  export PYCTI_VERSION=$pycti_default_version
+}
+
+load_env () {
+  export "$(grep -v '^#' "$1" | xargs -d '\n')"
+}
+
+if ! [ -d venv/ ]; then
+  ./initialize.sh
+  if ! [ $? ]; then
+    echo "Failed to install dependencies." >&2
+    exit 1
+  fi
+fi
+source venv/bin/activate
+
```

Comment:
is this venv still necessary?

Reply:
Removed venv activation as no more necessary,